requirements/docs.txt

```diff
@@ -66,3 +66,5 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 urllib3==2.0.2
     # via requests
+setuptools
+    # Necessary for pkg_resources to be available for python 3.12
```

Comment:
that's a generated file, and this comment looks manually-added. add both to `docs.in` instead and use `pip-compile` to regenerate this file.